src/rev_ai/models/asynchronous/translation_language_options.py

```diff
@@ -0,0 +1,53 @@
+from .summarization_job_status import SummarizationJobStatus
+from ..nlp_model import NlpModel
+
+"""Translation language request options."""
+
+
+class TranslationLanguageOptions:
+    def __init__(
+            self,
+            language: str = None,
+            model: NlpModel = None
```

Comment:
not sure whether you discussed it already., I see that some code is written in py2.x style without the type annotations, other - with..

while strict typing is good, wondering how can we be more consistent in the publicly available code

Reply:
Agreed to encourage incremental improvements going forward.